setup-buildengine.sh

```diff
@@ -392,12 +391,20 @@ while [[ ${1:-} ]]; do
         unregister ) shift
             OPT_DO_UNREGISTER=1
             ;;
+        --basename ) shift
+            OPT_DEFAULT_BASENAME=$1; shift
+            [[ -z $OPT_DEFAULT_BASENAME ]] && fatal "empty basename option given"
+            ;;
         * )
             usage quit
             ;;
     esac
 done
 
+SAILFISH_DEFAULT_TOOLING="${OPT_DEFAULT_BASENAME}-RELEASE"
+SAILFISH_DEFAULT_TARGETS="${OPT_DEFAULT_BASENAME}-RELEASE-i486
+                          ${OPT_DEFAULT_BASENAME}-RELEASE-armv7hl"
```

Comment:
I would prefer keeping the template near the beginning of the file, using "BASENAME" there and replacing it at this place with $OPT_TARGET_BASENAME, similarly to how RELEASE is replaced with $OPT_RELEASE